src/shared/components/RadioCardButton/RadioCardButton.styles.ts

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled'
+
+import { colors, sizes } from '@/shared/theme'
+
+import { Text } from '../Text'
+
+type RadioCardLabelProps = {
+  selected?: boolean
+  disabled?: boolean
+  error?: boolean
+}
+
+const getRadioCardBorderColor = ({ selected, disabled, error }: RadioCardLabelProps) => {
+  if (disabled) {
+    return colors.gray[500]
+  } else if (error) {
+    return colors.error
+  } else if (selected && !error) {
+    return colors.blue[500]
+  }
+  return colors.gray[500]
+}
+
+export const RadioCardLabel = styled.label<RadioCardLabelProps>`
+  padding: ${sizes(4)};
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  border: 1px solid ${({ selected, disabled, error }) => getRadioCardBorderColor({ selected, disabled, error })};
```

Comment:
You can just pass the function, it should get called with the props:
```suggestion
  border: 1px solid ${getRadioCardBorderColor};
```